docs/api/topics/ratings.rst

```diff
@@ -21,6 +21,11 @@ _`List`
 
     :query app: the ID or slug of the app whose ratings are to be returned.
     :query user: the ID of the user or `mine` whose ratings are to be returned.
+    :query lang: a language to filter ratings by if `match_lang` is set.
+    :query match_lang: a boolean to specify to match langauge or not. If unset
+                       all results are returned, if 1 only results matching
+                       `lang` are returned if 0 only results not matching
+                       `lang` are returned. All other values are ignored.
```

Comment:
It is essentially just there to maintain backwards compatibility. We are currently sending `region` and `lang` on each request from fireplace so we could get some weird behaviour if we start filtering on `lang`.